os_stub/cryptlib_mbedtls/rand/rand.c

```diff
@@ -74,7 +74,9 @@ boolean random_bytes(OUT uint8 *output, IN uintn size)
 
 int myrand(void *rng_state, unsigned char *output, size_t len)

```

Comment:
Please add a comment that myrand / f_rng is not documented, but that examination of the code revealed that a return value of 0 is considered a success, while a return value that is non-zero is considered a failure or error.